contrib/util/reset-ps-all-apps.sh

```diff
@@ -0,0 +1,12 @@
+apps=$(curl -H "Authorization: token $DEIS_TOKEN" http://$DEIS_SERVER/v1/apps  | jq -r '.results | map(.id) | join(" ")')
```

Comment:
Personally I'd rather `deis apps:list | grep -v "==="` because then the client can properly handle/display server errors.